public/src/components/PostComponent.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+    faAngleUp,
+    faAngleDown,
+    faCommentAlt,
+    faShare,
+    faBookmark,
+    faStar,
+} from '@fortawesome/free-solid-svg-icons';
+import methods from '../utils/methods';
+
+const {
+    getDate,
+    reformatTextToHtml,
+} = methods;
+
+export default class PostComponent extends React.Component {
+    constructor(props) {
+        super(props);
+        this.calculateDate = this.calculateDate.bind(this);
+    }
+
+    getPostImage() {
+        const { thumbnail, thumbnail_height, thumbnail_width } = this.props;
+        const imageWidth = thumbnail_width !== 'null' ? +thumbnail_width : 'auto';
+        const imageHeight = thumbnail_height !== 'null' ? +thumbnail_height : 'auto';
```

Comment:
You do `thumbnail !== 'null'` check on the next line, can `thumbnail_height` be `null` even if `thumbnail` is not `null`?

Reply:
you're right

Reply:
fixed https://github.com/YuriyBC/Reddit-archive/commit/890e5b07d9c75c92f8f0644051098393220d352f